src/Language/PureScript/CodeGen/JS/Common.hs

```diff
@@ -26,6 +27,13 @@ moduleNameToJs (ModuleName pns) =
 identToJs :: Ident -> Text
 identToJs (Ident name) = properToJs name
 identToJs (GenIdent _ _) = internalError "GenIdent in identToJs"
+identToJs UnusedIdent = C.__unused
+
+-- | Convert an 'Ident' into a valid JavaScript identifier, or Nothing.
```

Comment:
A function returning both a `Maybe` AND throwing exceptions is a red flag to me, which is almost guaranteed to be used wrong. And why do we still need the old version of this function?

Since this function is only used in a single place, I think it would be better to define it inline where it has more context.

Reply:
Agreed. Will fix.